l10n_it_fatturapa/data/fatturapa_data.xml

```diff
@@ -10,32 +9,32 @@
         <record id="fatturapa_RF02" model="fatturapa.fiscal_position">
             <field name="code">RF02</field>
             <field
-                name="name"
```

Comment:
Sai mica da che configurazione arrivano queste modifiche? A me sembrava più leggibile prima

Reply:
Vedi degli errori qui:
https://github.com/OCA/l10n-italy/actions/runs/3381763531/jobs/5616007369
li causa  pylint ma non mi chiedere come mai...

Reply:
Ah ho trovato, in pratica invece di spostare a sinistra i parent che sono indentati di 8 spazi ma dovrebbero essere indentati solo di due, sposta solo questi attributi.
Ti ho proposto la modifica in https://github.com/TheMule71/l10n-italy/pull/29

Reply:
L'indentazione a 8 deriva dall'eliminazione di `<data>`. Speravo di minimizzare il diff, facendo vedere solo quello che cambiava ai fini migrazione. In realtà si è messo di mezzo pylint e il risultato finale è così così.
La tua PR va benissimo avevo già in mente di spostare tutto in un commit separato.

Se mi gira potrei spezzare i commit e farne uno --no-verify, e solo dopo reindentare tutto.